pixart.js

```diff
@@ -1 +1,18 @@
+$(document).ready(function() {
+  $("#set-color").on("click", function(evt) {
+    evt.preventDefault();
+    var newColor = $("#color-field").val();
+    $(".brush").css("background", newColor)
+  })
 
+  var divSq = []
+  for (var i = 0; i < 8000; i++) {
+    divSq.push('<div class="square"></div>');
+  }
+  $("body").append(divSq);
+
+  $(".square").on("mouseover", function(){
+    var newColor = $("#color-field").val();
```

Comment:
This works, but is not quite as robust. If I 'set' a color (by clicking the set button), then clear the input field, this code won't use the color that I set, as it is still pulling from the input field.

Instead, I'd do:

var color = $(".brush").css("background-color");
which gets the value that was 'set'.